references/cloud-logging-shared-flow/sharedflowbundle/policies/SC.CloudLogging.xml

```diff
@@ -0,0 +1,75 @@
+<!--
+Copyright 2021 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<ServiceCallout name="SC.CloudLogging">
+    <Request>
+        <IgnoreUnresolvedVariables>true</IgnoreUnresolvedVariables>
+        <Set>
+            <Payload contentType="application/json">{
+    "logName": "projects/{organization.name}/logs/apigee-runtime",
+    "resource" : {
+      "type": "api",
+      "labels": {}
+    },
+    "labels": {
+        "application": "apigee"
+    },
+    "entries": [{
+        "severity": "INFO",
+        "jsonPayload": {
+            "organization": "{organization.name}",
+            "environment": "{environment.name}",
+            "apiProxy": "{apiproxy.name}",
+            "apiProxyRevision": "{apiproxy.revision}",
+            "apiProduct": "{apiproduct.name}",
+            "developerApp": "{apiproduct.name}",
+            "clientId": "{client_id}",
+            "developerId": "{developer.id}",
+            "requestUri": "{request.uri}",
+            "requestUrl": "{request.url}",
+            "verb": "{request.verb}",
+            "correlationId": "{messageid}",
+            "proxyRequestReceived": "{client.received.end.timestamp}",
+            "proxyResponseSent": "{client.sent.end.timestamp}",
+            "targetResponseReceived": "{target.received.end.timestamp}",
+            "targetRequestSent": "{target.sent.end.timestamp}",
+            "targetResponseCode": "{message.status.code}",
+            "proxyResponseCode": "{response.status.code}",
+            "faultName":"{fault.name}"
+        }
+    }],
+    "partialSuccess": true
+  }
+</Payload>
+            <Verb>POST</Verb>
+        </Set>
+    </Request>
+
+    <HTTPTargetConnection>
+        <Authentication>
+            <GoogleAccessToken>
+                <Scopes>
+                    <Scope>https://www.googleapis.com/auth/cloud-platform</Scope>
```

Comment:
A more granular scope may be more appropriate `https://www.googleapis.com/auth/logging.write`
(per https://developers.google.com/identity/protocols/oauth2/scopes#logging)

Reply:
I see your point. I decided to follow 

```sh
As a result, a best practice is for you to set the full cloud-platform access scope on the instance, then control your service account's access by granting IAM roles to the service account. See Service account permissions for details.
```

https://cloud.google.com/compute/docs/access/service-accounts, any strong opinions for either?

Reply:
No strong opinions for either.

Reply:
What is the disadvantage of putting `logging.write` here as the scope?

Reply:
No direct disadvantage but the practice to have minimal service accounts (basically 1 per use case) and therefore the scope isn't needed (RBAC = scope). I prefer this approach because a) it also works for gRPC in ExternalCallout policy and b) changes can be audited in the audit log whilst scope changes in the proxy bundle are not explicitly highlighted.

Reply:
There is no doubt that minimal service accounts are a must-have. However, concantrating on the scope element for the moment, isn't it a best practice for tokens to contain the exact scopes that they require for the action that they will be performing (last paragraph of https://developers.google.com/identity/protocols/oauth2#2.-obtain-an-access-token-from-the-google-authorization-server). We know that this token will be used for writing to the logs - nothing else.

I propose that we put `logging.write` explicitly here to assert this best practice - unless you see any disadvantages of this. This is additional to the service account creation and RBAC, etc applied to that service account - which are outside the scope of the line we are talking about here.

Am I missing something?